cves/kernel/CVE-2016-3137.yml

```diff
@@ -49,13 +49,13 @@ description_instructions: |
 
   Try to still be specific in your description, but remove project-specific
   stuff. Remove references to versions, specific filenames, and other jargon
-  that outsiders to this project would not understand. Technology like "regular
-  expressions" is fine, and security phrases like "invalid write" are fine to
+  that outsiders to this project would not understand. Technology like 'regular
+  expressions' is fine, and security phrases like 'invalid write' are fine to
   keep too.
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: 'This vulnerability was caused because of the failure to include a check for the existance/non-existance of an interupt-out value. This allowed physically present attackers to insert a properly formatted USB that rquires cypress_m8 drivers. Once the USB is inserted the kernel would then dereference a null pointer causing a crash.'
```

Comment:
typo in requires. You have "rquires." Small gramatical thing, there should be a comma after "Once the USB is inserted" (i.e. Once the USB is inserted, the kernel would...)